Code/Editor/ViewportTitleDlg.cpp

```diff
@@ -953,13 +953,13 @@ void CViewportTitleDlg::CheckForCameraSpeedUpdate()
 void CViewportTitleDlg::OnGridSnappingToggled()
 {
     m_gridSizeActionWidget->setEnabled(m_enableGridSnappingAction->isChecked());
-    MainWindow::instance()->GetActionManager()->GetAction(ID_SNAP_TO_GRID)->trigger();
+    MainWindow::instance()->GetActionManager()->GetAction(AzToolsFramework::SnapToGrid)->trigger();
 }
 
 void CViewportTitleDlg::OnAngleSnappingToggled()
 {
     m_angleSizeActionWidget->setEnabled(m_enableAngleSnappingAction->isChecked());
-    MainWindow::instance()->GetActionManager()->GetAction(ID_SNAPANGLE)->trigger();
+    MainWindow::instance()->GetActionManager()->GetAction(AzToolsFramework::SnapAngle)->trigger();
 }
```

Comment:
As a follow-up change after this PR I wonder if we can get rid of these functions too, they might not actually be needed anymore... I can't quite recall how/when they're called but we might be able to simplify the logic here...

Reply:
Yeah totally, it just reminded me and I thought I'd mention it 🙂